sstables/mx/reader.cc

```diff
@@ -1523,7 +1527,7 @@ class mx_sstable_mutation_reader : public mp_row_consumer_reader_mx {
 
             if (!present) {
                 _sst->get_filter_tracker().add_false_positive();
-                co_return;
+                co_return false;
```

Comment:
An even better interface is to return optional\<something>. It can be a method of sstable itself. nullopt means we don't need the something because we won't be reading anything. A something (reader? something used to initialize the reader?) can be used to actually to the reading.

But, that's for another day.